plugins/github-pull-requests-board/src/api/useOctokitGraphQl.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Octokit } from '@octokit/rest';
+import {
+  useApi,
+  githubAuthApiRef,
+  configApiRef,
+} from '@backstage/core-plugin-api';
+import { readGitHubIntegrationConfigs } from '@backstage/integration';
+
+let octokit: any;
+
+export const useOctokitGraphQl = <T>() => {
+  const auth = useApi(githubAuthApiRef);
+  const config = useApi(configApiRef);
+
+  const baseUrl = readGitHubIntegrationConfigs(
+    config.getOptionalConfigArray('providers.github') ?? [],
```

Comment:
I believe the config we are looking for is going to be under the `integrations.github`
```suggestion
    config.getOptionalConfigArray('integrations.github') ?? [],
```